src/services/httpClient.js

```diff
@@ -69,7 +69,7 @@ export class HttpClient {
     const request = HttpClient.createAjaxRequest({ ...options, url }, withAuth);
     return Observable.ajax(request)
       // If success, extract the response object and enforce camelCase keys
-      .map(({ response }) => camelcaseKeys(response, { deep: true }))
+      .map(({ response }) => response && camelcaseKeys(response, { deep: true }))
```

Comment:
Correct. Delete didn't return a response body at all.

Reply:
I see. Instead of having the stream return an undefined value let's do this instead.

`camelcaseKeys((response || {}), { deep: true })`

That way we will just get an empty object instead of nothing.